common.blocks/menu/menu.bh.js

```diff
@@ -31,6 +37,8 @@ module.exports = function(bh) {
                     var i = 0, itemOrGroup;
                     while(itemOrGroup = content[i++]) {
                         if(itemOrGroup.block === 'menu-item') {
+                            itemOrGroup.attrs = (itemOrGroup.attrs || {});
+                            itemOrGroup.attrs.id = itemOrGroup.attrs.id || ctx.generateId(); // Support a11y
```

Comment:
@narqo Each call uses precalculated `_expandoId` (date) and numeric `lastGenId`. It means no. It returns unique id for each BH instance and each generateId call.